infoenergia_api/contrib/contracts.py

```diff
@@ -136,6 +136,22 @@ def tertiaryPower(self):
          'P{}'.format(i): int(period['potencia'] * 1000) for i, period in enumerate(period_powers, 1)
         }
 
+    @property
+    def tertiaryPowerHistory(self):
+        test = [
```

Comment:
as the best name for this variable is `tertiary_power_history`, i think that a simple `return` here is fine